metpy/calc/tests/test_thermo.py

```diff
@@ -413,6 +415,15 @@ def test_wet_psychrometric_rh_kwargs():
     assert_almost_equal(psychrometric_rh, 82.9701 * units.percent, 3)
 
 
+def test_mixing_ratio_from_relative_humidity():
+    """Tests relative humidity from mixing ratio."""
```

Comment:
By convention, the first line of the doc string should be imperative, so "Test" rather than "Tests". I see you just copied that from the rest of the file--we need to go back and fix those (Unless you're interested in doing so...)

Reply:
Sounds good. I would have no problem updating all the other first-lines as well. Would you prefer that to be in this PR or a new one?

Reply:
Here is fine. Thanks!